src/components/pages/MentorMenteeMatching/MentorMenteeInfo.js

```diff
@@ -0,0 +1,131 @@
+import React, { useState } from 'react';
+import { Card } from 'antd';
+import { EditOutlined } from '@ant-design/icons';
+import { Input } from 'antd';
+
+const MentorMenteeInfo = () => {
+  const [edit, setEdit] = useState(false);
+
+  const fakeData = [
+    {
+      information: {
+        email: 'random@gmail.com',
+        phone: '123-123-123',
+        random: 'random',
+      },
+      Mentors: {
+        first: 'Jimmy Bold',
+      },
+    },
+  ];
+
+  return (
+    <>
+      {fakeData.map(data => (
+        <div
+          style={{
+            display: 'flex',
+            flexDirection: 'row',
+            justifyContent: 'space-evenly',
+          }}
+        >
+          <Card title="Information" style={{ width: '30%' }}>
+            <p>{data.information.email}</p>
+            <p>{data.information.phone}</p>
+            <p>{data.information.random}</p>
+          </Card>
+          <Card
+            title="Mentors"
+            style={{
+              width: '30%',
+            }}
+            extra={<EditOutlined onClick={() => setEdit(!edit)} />}
+          >
+            {edit ? (
+              <div style={{ width: '80%' }}>
+                <Input
+                  placeholder="Assign Mentor"
+                  value={
+                    data.Mentors.first ? data.Mentors.first : 'Assign Mentor'
+                  }
+                />
+              </div>
+            ) : (
+              <div style={{ width: '80%' }}>
+                <p>
+                  {!data.Mentors.first ? (
+                    <Input placeholder="Assign Mentor" className="mentors" />
```

Comment:
This is a small thing, but worth noting - it looks like you have some inconsistent quotation marks in this file, with some above this being singles, while this is using doubles. It might seem like something you don't need to worry about, but when you get hired these are some of the things your future company may actually be looking for!